src/fetcher.py

```diff
@@ -64,10 +64,14 @@ def __fetch_paginated(self, url: str) -> list[dict]:
 
 def parse_link_header(link_header: str) -> dict[str, str]:
     """Parse the `Link` header from a GitHub API response into a dictionary"""
-    # TODO: implement
-    return {
-        "last": "https://api.github.com/repositories/198065251/commits?per_page=1&page=3095"
-    }
+    chunks = link_header.split(",")
+    output = {}
+    for i in range(0, len(chunks)):
+        link = chunks[i][chunks[i].index("<") + 1 : chunks[i].index(">")]
+        name = chunks[i][chunks[i].index("\"") + 1 : chunks[i].index("\"", chunks[i].index("\"") + 1)]
```

Comment:
In my experience, string parsing like this tends to be extremely brittle. This would be easier to maintain using a regex instead.

Reply:
Ah, I hadn't thought of using regex for something like this. I'll try that next time